api/src/services/webhooks.ts

```diff
@@ -1,32 +1,38 @@
+import { ErrorCode, createError } from '@directus/errors';
 import type { Bus } from '@directus/memory';
 import { useBus } from '../bus/index.js';
-import type { AbstractServiceOptions, Item, MutationOptions, PrimaryKey, Webhook } from '../types/index.js';
+import { useLogger } from '../logger.js';
+import type { AbstractServiceOptions, MutationOptions, PrimaryKey, Webhook } from '../types/index.js';
 import { ItemsService } from './items.js';
 
+const logger = useLogger();
+
 export class WebhooksService extends ItemsService<Webhook> {
 	messenger: Bus;
 
 	constructor(options: AbstractServiceOptions) {
 		super('directus_webhooks', options);
 		this.messenger = useBus();
+
+		logger.warn(
+			'Webhooks are deprecated and the WebhooksService will be removed in an upcoming release. Creating/Updating Webhooks is disabled, use Flows instead',
+		);
 	}
 
-	override async createOne(data: Partial<Item>, opts?: MutationOptions): Promise<PrimaryKey> {
-		const result = await super.createOne(data, opts);
-		this.messenger.publish('webhooks', { type: 'reload' });
-		return result;
+	override async createOne(): Promise<PrimaryKey> {
+		throw new (createError(ErrorCode.MethodNotAllowed, 'Webhooks are deprecated, use Flows instead', 405))();
```

Comment:
Can we create this error once and re-use it perhaps?

Reply:
Didnt deem it necessary since this whole file is gonna get nuked anyway but theoretically makes sense of course, added in https://github.com/directus/directus/pull/21808/commits/54070c214d48e4a8eefd49b08d83c56ef039a362